src/Level/Player.java

```diff
@@ -105,51 +284,59 @@ protected void playerWalking() {
             keyLocker.lockKey(INTERACT_KEY);
             map.entityInteract(this);
         }
-
+        moveAmountX = 0;
+        moveAmountY = 0;
         // if walk up key is pressed, move player up
-        if (Keyboard.isKeyDown(MOVE_UP_KEY)) {
+        if (Keyboard.isKeyDown(MOVE_UP_KEY) || Keyboard.isKeyDown(Key.W)) {
```

Comment:
No need to double check if key is W

private static Key MOVE_LEFT_KEY = Key.A; // Left Movement for A
    private static Key MOVE_RIGHT_KEY = Key.D; // Right Movement for D
    private static Key MOVE_UP_KEY = Key.W; // Up Movement for D
    private static Key MOVE_DOWN_KEY = Key.S; // Down Movement for S